bugwarrior/db.py

```diff
@@ -89,7 +95,7 @@ def hamdist(str1, str2):
 
 def get_managed_task_uuids(tw, key_list, legacy_matching):
     expected_task_ids = set([])
-    for keys in key_list.values():
+    for keys in list(key_list.values()):
```

Comment:
I'm not knowledgeable enough on Python to criticize view, I'm just trying to make sense of changes introduced by futurize in order to answer your objections :-)
The example above is only to show that the same source code can expose very different behavior depending on the version of Python used, not to criticize the behaviors.